songs/159_I_Love_Punsch.md

```diff
@@ -0,0 +1,16 @@
+---
+title: I Love Punsch
+author: Samuel Larsson (2020)
+melody: Java Jive
+tags: [punsch, eng]
+---
+
+I love punsch in bites of three
+I love the punsch bottle and it loves me
+Arrack and tea, makes the punsch just for me
```

Comment:
```suggestion
Arrack and tea, make the punsch just for me
```

Reply:
Here it is implied that the arrack and tea are actively creating the punsch, so this is as intended